source/connection-troubleshooting.txt

```diff
@@ -0,0 +1,242 @@
+.. _golang-connection-troubleshooting:
+
+================================
+Connection Troubleshooting Guide
```

Comment:
Q: could we just call this connection troubleshooting? many pages are "guides" but we omit that word from the titles